docs/source/starter/expert_installation.rst

```diff
@@ -14,7 +14,7 @@
 Expert User Installation (Administrative privileges)
 ****************************************************
 
-The `Complete Local conda-based GeoIPS Installation <./installation.rst>`
+The :ref:`Complete Local conda-based GeoIPS Installation<installation>`
```

Comment:
I guess these links were originally just set up to link within the github rendered RST pages, and were broken links within the html documentation?

Reply:
Yeah, they didn't even work rendered directly within github...